recipes-ni/fw-printenv/files/fw_printenv

```diff
@@ -23,41 +23,46 @@ File syntax example:
 In this case, var1 will be deleted, var2 and other_value
 will be set with the values passed. The value can contain any
 number of spaces.
+
+Variables defined in multiple places will have the
+following preference order: BIOS -> EFI -> GRUB
```

Comment:
This seems like the inverse ordering from what I would expect. EFI is a more recent and featureful technology than BIOS, and I would expect it to be preferred. And the GRUB install has a more-specific scope than BIOS/EFI, so I would expect it to be preferred over all.

I understand there was a previous behavior on ARM to lock the uboot variables and consider them canonical. But I don't see the reason that we would carry that behavior to x64.

Can we reverse the ordering?

Reply:
During a team discussion we decided this might inadvertently change behavior client teams were depending on.  I therefore think we should keep the order as it exists until we make a larger change to source these variables in a direct way from their data sources.